sequencer/api/config.toml

```diff
@@ -6,4 +6,4 @@ DOC = "Get the Hotshot configuration for the current node."
 [route.env]
 PATH = ["/env"]
 METHOD = "GET"
-DOC = "Get all ESPRESSO environment variables set for the current node."
\ No newline at end of file
+DOC = "Get all ESPRESSO environment variables defined in public-env-vars.toml file for the current node"
```

Comment:
I think the documentation is fine the way it was before. This will be exposed publicly in the auto-generated docs page, and users of this API won't necessarily know what `public-env-vars.toml` is